src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SubMultipleServiceTest.java

```diff
@@ -130,7 +137,8 @@ public void populateFilterDefaultedAllDynamicListLogic() {
         String result = "DynamicFixedListType(value=DynamicValueType(code=999999, label=Select All), " +
```

Comment:
A few checkstyle issues missed on this file. Not just line longer than 120 characters but also '+' on new line and distance between varaibles.